src/App.js

```diff
@@ -1,23 +1,20 @@
-import './App.css';
+import React from 'react';
+import { Routes, Route } from 'react-router-dom';
+import Greeting from './components/greeting';
+
+// const App = () => (
+//   <Router>
+//     <Routes>
+//       <Route exact path="/" element={<Greeting />} />
+//     </Routes>
+//   </Router>
+// );
```

Comment:
- [ ] Kindly avoid leaving commented pieces of code in your codebase. It makes your codebase unclean and is not a good practice :+1: